### PR TITLE
Add inactive member

### PR DESCRIPTION
**Pull request by idujin**

### CODE REVIEW

File: csv_editor.py [hunk 1]
```diff
@@ -129,7 +129,8 @@ def daily_check_in_auto(self, users):
         if(is_working_day):
             yy, mm, dd = current_time.year,current_time.month,current_time.day
             for user in users:
-
+                if user in inactive_user:
```
Comment (yhbyhb) [inline csv_editor.py]: It's better removing inactive_user in users rather than checking inactive_user in for loop.

File: csv_editor.py [hunk 2]
```diff
@@ -91,18 +91,18 @@ def get_display_csv(self, user):
         month = datetime.now().date().month
         return df[np.logical_and(df['year']==year, df['month']==month)]
 
-    def get_current_score_dataframe(self, users, except_user = None):
+    def get_current_score_dataframe(self, users, inactive_user = None):
```
Comment (yhbyhb) [inline csv_editor.py]: how about inactive_user"s"?

File: constants.py [hunk 3]
```diff
@@ -1 +1,2 @@
-USERS = ["유진", "애리", "지은"]
\ No newline at end of file
+USERS = ["유진", "애리", "지은"]
+INACTIVE = ["애리"]
```
Comment (yhbyhb) [inline constants.py]: how about use set instead of list on both `USERS` and `INACTIVE_USERS`?

**Review by yhbyhb — COMMENTED** [review]
You can do better! :-)

File: csv_editor.py (hunk 1 above)
Comment (idujin) [reply csv_editor.py]: Done!

File: csv_editor.py (hunk 2 above)
Comment (idujin) [reply csv_editor.py]: Done

File: constants.py (hunk 3 above)
Comment (idujin) [reply constants.py]: Done

**Review by yhbyhb — APPROVED** [review]

**Pull request merged**